misc/dataclasses-json/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "dataclasses-json" %}
```

Comment:
we don't actually need this, since it's available on conda-forge:
https://anaconda.org/conda-forge/dataclasses-json

Reply:
Done